package.json

```diff
@@ -16,11 +16,13 @@
     "build": "vue-cli-service build --target lib --name vue-money-amount --inline-vue src/MoneyAmount.vue",
     "test:unit": "vue-cli-service test:unit",
     "test:unit:watch": "vue-cli-service test:unit --watch",
-    "lint": "vue-cli-service lint --fix"
+    "lint": "vue-cli-service lint --fix",
+    "storybook": "start-storybook -s ./public -p 6006"
```

Comment:
Can you please add a storybook build and add it to the github actions as a build step?
We want to be sure we didn't break the storybook.

Reply:
Ok, I think it's good if you want to check!